sysutils/glances/Portfile

```diff
@@ -20,9 +20,9 @@ long_description    Glances is a cross-platform monitoring tool which aims to pr
 
 platforms           {darwin any}
 
-checksums           rmd160  edae9bbac33ceedca7ec20151ad0281dcb80c12a \
-                    sha256  aba3cf2e8a16cac37119d687b1a8d8689b60d5ae815e9ef336a66352d811562e \
-                    size    7036471
+checksums           rmd160  2322867040c6c21e701ac0d01c54f132b00f7e88 \
+                    sha256  473793bf118a0c34ef54bc7004cfc2d7bbc8752551b440e07ed37684a68d50b3 \
+                    size    7041565
 
 python.default_version \
                     312
```

Comment:
this should ideally be using Python 3.13 as that is MacPorts' default. Any reason why it's not?

Reply:
Python 3.13 should work.  Let e do some testing first.  

If python 3.13 works, do I change the `python.default_version` statement in the Portfile or do I just delete it?

Reply:
Compiling using python 3.13 does not work....

Dependency `py313-future` not found.....   Must use python 3.12 since `py312-future` exists in MacPorts.

Reply:
P.S. I do not know why py-future has not been updated to include python 3.13. Let me update py-future and see what happens.....

Reply:
I updated the port for `py-future` for python 3.13 and was able to successfully build `glances` using Python 3.13.  'glances` runs just fine.  It looks like I will need to submit another PR for `py-future`

Reply:
This port will not build until this PR for `py-future` is merged:

https://github.com/macports/macports-ports/pull/28216

Reply:
FYI -- the latest push includes your suggested change.